docs/index.md

```diff
@@ -27,6 +27,7 @@ For more granular packages we recommend reading the documentation.
 * Automate Azure Key Vault tasks ([powershell](features/powershell/azure-key-vault))
```

Comment:
This change should be done in the `./preview` folder, not in the published folder.